src/components/tips/tip-card.tsx

```diff
@@ -100,108 +119,40 @@ const TipCard: React.FC<{tip: Tip; i: number}> = ({tip, i}) => {
                 />
               </div>
             )}
-            <span className="text-left dark:text-indigo-100 text-gray-700 lg:text-sm text-[0.75rem] opacity-80 leading-none">
+            <span className="text-left  lg:text-sm text-[0.75rem] opacity-80 leading-none">
               <span className="sr-only">{tip.title} by </span>
-              {tip?.instructor?.name}
+              <Link
+                href={`q/resources-by-${tip?.instructor?.slug}`}
+                className="hover:underline"
+              >
+                {tip?.instructor?.name}
+              </Link>
             </span>
           </div>
+          <TagList tags={tip.tags} />
           {tip?.duration && (
-            <span className="text-xs dark:text-indigo-100 text-gray-700">
-              {convertTimeWithTitles(Math.floor(tip.duration), {
-                showSeconds: true,
-              })}
-            </span>
+            <div className="flex items-center gap-1">
+              <ClockIcon className="w-4 h-4 opacity-60" />
+              <span className="text-xs">
+                {convertTimeWithTitles(Math.floor(tip.duration), {
+                  showSeconds: true,
+                })}
+              </span>
+            </div>
           )}
-          <TagList tags={tip.tags} />
         </div>
         {tip?.description && (
-          <ReactMarkdown className="prose dark:prose-dark dark:prose-dark-sm dark:prose-a:text-blue-300 prose-a:text-blue-500 prose-sm line-clamp-2 max-w-[50ch] pt-2">
+          <ReactMarkdown
+            className="prose dark:prose-dark dark:prose-dark-sm dark:prose-a:text-blue-300 prose-a:text-blue-500 prose-sm line-clamp-2 pt-2"
+            components={{
+              a: (props) => <span>{props.children}</span>,
+              p: (props) => <p className="text-balance">{props.children}</p>,
+            }}
```

Comment:
didn't know about the components prop for `ReactMarkdown` but this gives you control of how different elements are presented when converted to html. Specifically.. we don't want links in the markdown to actually render because it was causing hydration errors also `text-balance` on description is nice.